chord_metadata_service/experiments/cleanup.py

```diff
@@ -0,0 +1,31 @@
+# TODO
+
+from chord_metadata_service.cleanup.remove import remove_not_referenced
+from chord_metadata_service.utils import dict_first_val
+from .models import Experiment, ExperimentResult, Instrument
+
+__all__ = [
+    "clean_experiment_results",
+    "clean_instruments",
+]
+
+
+# TODO: Remove this when we have one-to-many ?
+def clean_experiment_results() -> int:
+    results_referenced = set()
+
+    # Collect references to results from experiments
+    results_referenced.update(map(dict_first_val, Experiment.objects.values("experiment_results__id")))
```

Comment:
There is a double '_' in the string "experiment_results__id", not a typo?

Reply:
not a typo, that's the Django version of `experiment_results.id` - but it does remind me I should test experiments a bit more thoroughly